src/patternfly/components/AppLauncher/examples/AppLauncher.md

```diff
@@ -328,7 +328,7 @@ import './AppLauncher.css'
 | `.pf-c-app-launcher__group-title` | `<h1>` | Defines a title for a group of items. |
 | `.pf-c-app-launcher__menu-item` | `<a>`, `<button>` | Defines a menu item. |
 | `.pf-c-app-launcher__menu-item-icon` | `<span>` | Defines the wrapper for the menu item icon. |
-| `.pf-c-app-launcher__menu-item-external-icon` | `<i>` | Defines the external link icon that appears on hover/focus. Use with `.pf-m-external`. |
+| `.pf-c-app-launcher__menu-item-external-icon` | `<span>` | Defines the external link icon that appears on hover/focus. Use with `.pf-m-external`. |
```

Comment:
Can you match with the working in the previous line? Something like "Defines the wrapper for the external link icon that appears on hover/focus. Use with `.pf-m-external`."

Reply:
OK